api.py

```diff
@@ -57,3 +57,10 @@ def divik(dataset_id: int):
         return load.divik_config(dataset_id, divik_id)
     # unused for a while, informs that result was requested
     return as_response(load.divik_result(dataset_id, divik_id))
+
+
+@app.route('/heatmap/<int:dataset_id>')
+@missing_key_returns_404
+def heatmap(dataset_id: int):
+    channel_id = require('channelId', int)
```

Comment:
In c# version there was also field flag. Is it necessary or it can be removed from angular heatmap service URL?

Reply:
It was used only for some kind of distinction between different endpoints. If you provide it here, there is no harm - I've checked it already. Functionally, for you, nothing should change.